introduction/quickstart.md

```diff
@@ -0,0 +1,290 @@
+
+```shell
+curl https://glaredb.com/install.sh | sh
+
+./glaredb
+```
+
+2. Start writing SQL! You can try out the below to query a Parquet file hosted
+on GitHub over HTTP. This file contains NYC real estate transactions from 
+January 2022.
+
+```sql
+SELECT * 
+FROM read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet');
+```
+
+3. Download a file and query it locally. [Here is a file with the next month
+of transaction data.] After downloading it, query it using its relative or 
+absolute path. 
+
+```sql
+// Relative path to the same directory from which you started the GlareDB CLI
+SELECT * FROM read_parquet('./f55363e2587849bcb25c057be706c69d-0.parquet');
+
+// Absolute path to the file
+SELECT * FROM read_parquet('/Users/username/Downloads/f55363e2587849bcb25c057be706c69d-0.parquet');
+```
+
+4. Query a remote Postgres database using the `read_postgres()`
+function:
+
+```sql
+SELECT * FROM read_postgres(
+  'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres', --connection
+  'public', --schema name
+  'customer' --table name
+);
+```
+
+5. Join between the local Parquet file and the remote Postgres database to
+get a count of number of sales per NYC borough, and include the names of the
+NYC borough where the sales took place:
+
+```sql
+SELECT 
+    COUNT(sales.sale_date),
+    lookup.borough_name
+FROM 
+   read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet') sales
+JOIN 
+    read_postgres(
+        'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres', 
+        'public',
+        'borough_lookup'
+    ) lookup
+ON sales.borough = lookup.borough_id
+GROUP BY lookup.borough_name;
+```
+
+6. Save your data for easy access later using the previous query and `COPY TO`:
+TODO: This currently errors. Remove if not quickly resolved.
+```sql
+COPY (
+    SELECT 
+        COUNT(sales.sale_date),
+        lookup.borough_name
+    FROM 
+        read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet') sales
+    JOIN 
+        read_postgres(
+            'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres', 
+            'public',
+            'borough_lookup'
+        ) lookup
+    ON sales.borough = lookup.borough_id
+    GROUP BY lookup.borough_name
+) TO './my_joined_data.parquet';
+
+SELECT * 
+FROM './my_joined_data.parquet';
+```
+
+7. If you're using [GlareDB Cloud], create new tables that will 
+persist in Cloud storage using:
+
+```sql 
+CREATE TABLE my_new_table AS 
+SELECT * 
+FROM read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet');
+
+SELECT * 
+FROM my_new_table;
+```
+
+
+## Quickstart with SQL using Python
```

Comment:
```suggestion
## Quickstart with SQL using Python and Jupyter
```

Maybe, since we have some jupyter stuff at the beginning here.

Or we could remove the jupyter setup stuff and assume the user already has a way of running python that works for them.